amulet/api/world/world.py

```diff
@@ -94,11 +58,12 @@ def __init__(
         self._biome_palette = BiomeManager()
         self._biome_palette.get_add_biome("universal_minecraft:plains")
 
-        self._chunk_cache: ChunkCache = {}
-        shutil.rmtree(self._temp_directory, ignore_errors=True)
-        self._chunk_history_manager = ChunkHistoryManager(
-            os.path.join(self._temp_directory, "chunks")
+        self._chunk_cache2: ChunkManager = ChunkManager(
```

Comment:
chunk_cache2 isn't a really good name. Maybe chunk_manager or chunk_cache_manager?

Reply:
Yeh good point. The previous attribute had this name so I made a new one while I implemented the new one. I think there might also be another variable I need to remove